server/src/main/java/io/crate/auth/AuthSettings.java

```diff
@@ -44,6 +44,14 @@ private AuthSettings() {
         "auth.host_based.config.", Setting.Property.NodeScope
     );
 
+    public static final Setting<String> AUTH_HOST_BASED_JWT_ISS_SETTING = Setting.simpleString(
+        "auth.host_based.jwt.iss", Setting.Property.NodeScope
```

Comment:
Shouldn't they be marked as `exposed`? 
```suggestion
        "auth.host_based.jwt.iss", Setting.Property.NodeScope, Setting.Property.Exposed
```